tests/keras/activations_test.py

```diff
@@ -79,6 +79,16 @@ def test_softmax_invalid():
         f = K.function([x], [activations.softmax(x)])
 
 
+def test_softmax_3d():
+    """Test using a reference implementation of softmax.
+    """
+    x = K.placeholder(ndim=3)
+    f = K.function([x], [activations.softmax(x, axis=1)])
+    test_values = get_standard_values()[:, :, np.newaxis].copy()
+
+    result = f([test_values])[0]
```

Comment:
Please add a correctness check on `result` values. You can 1) use random values for `test_values`, 2) use `np.exp`/etc to compute the expected results.

Reply:
Wouldn't that be redundant since I've already added the correctness tests of the `softmax` operation in the `backend_test.py`  in this PR?

Reply:
It's cheap, and most other tests in this file have at least basic correctness checks. You could just check sums to 1.

Reply:
Ok sure thing.